templates/myuploads.html

```diff
@@ -0,0 +1,32 @@
+{% extends 'base-deprecated.html' %}
+{% load static %}
+{% load upToCharacter %}
+{% block content %}
+
+<h3>{{user.username | upToCharacter:"@"}}'s Uploads</h3>
```

Comment:
Hi, I'm back. For tonight, let's just start with this comment, and I'll try to actually run these locally tomorrow.
What do you think about "Your Syllabi"? Too vague?

Reply:
> I'll try to actually run these locally tomorrow.

Or, of course, I could still be waiting for the power to come back on after Isaias.

Reply:
Yikes, hope you're well!

I considered "Your Uploads" but I felt like that might've been redundant since the section with uploads (line 11) says "Your x uploaded syllabi". That section is second instead of first because I wanted the "Need to correct an error?" section at the top for visibility. I've been hoping to think of a better way but I haven't come up with anything yet